src/Compilers/CSharp/Test/Symbol/DocumentationComments/DocumentationModeTests.cs

```diff
@@ -19,16 +19,16 @@ public void XmlSyntaxError_Inline()
         {
             var xml = @"<unclosed>";
 
-            var expectedText = @"
+            var expectedText = $@"
 <?xml version=""1.0""?>
 <doc>
     <assembly>
         <name>Test</name>
     </assembly>
     <members>
-        <!-- Badly formed XML comment ignored for member ""T:Partial"" -->
-        <!-- Badly formed XML comment ignored for member ""T:Parse"" -->
-        <!-- Badly formed XML comment ignored for member ""T:Diagnose"" -->
+        {string.Format(CSharpResources.IDS_XMLIGNORED, "T:Partial")}
```

Comment:
>{string.Format(CSharpResources.IDS_XMLIGNORED, "T:Partial")} [](start = 8, length = 60)

Please change UI culture for the duration of this function instead. using (new EnsureEnglishUICulture())